PocketKit/Sources/PocketKit/MyList/SavesContainerViewController.swift

```diff
@@ -389,6 +389,15 @@ extension SavesContainerViewController {
             return
         }
 
+        collection.events.sink { [weak self] event in
```

Comment:
should we receive this on `DispatchQueue.main`?

Reply:
yes!